test/handle_test.cpp

```diff
@@ -39,7 +39,7 @@
 #include "get_handle.hpp"
 #include <vector>
 #include <thread>
-#include "test.hpp"
+#include "test_macro.hpp"
```

Comment:
Any way to resolve the "FAIL" problem without touching existing tests?

Reply:
@atamazov another way is only separate FAIL in to a new header file or remove this definition for now if no code is currently using it.

Reply:
The PR can be simplified, if we just remove the FAIL macro if its not being used anywhere

Reply:
> The PR can be simplified, if we just remove the FAIL macro if its not being used anywhere

@JehandadKhan done
cc @atamazov